plugins/techdocs/src/reader/components/Reader.tsx

```diff
@@ -585,16 +600,16 @@ export const useTechDocsReaderDom = (
                 position: static;
                 padding-left: 0;
               }
-              .md-footer-nav__link {
+              .md-footer__link, .md-footer-nav__link {
                 /* footer links begin to overlap at small sizes without setting width */
                 width: 50%;
               }
             }
 
             @media screen and (max-width: 600px) {
               .md-sidebar--primary {
-                left: -12.1rem !important;
-                width: 12.1rem;
+                left: -16rem !important;
+                width: 16rem;
               }
```

Comment:
Found an issue with the nav on mobile: it gets cut off by the Backstage side nav (which is at the bottom on mobile view). Including the following style here seems to fix it:

```
.md-sidebar--primary .md-sidebar__scrollwrap {
  bottom: ${sidebarConfig.mobileSidebarHeight}px;
}
```

Reply:
Thanks again, fixed 🙌🏻